android/src/main/java/com/facebook/flipper/plugins/inspector/descriptors/utils/stethocopies/FragmentCompat.java

```diff
@@ -82,20 +88,22 @@ public static FragmentCompat getSupportLibInstance() {
     @Nullable
     @Override
     public List<FRAGMENT> getAddedFragments(FRAGMENT_MANAGER fragmentManager) {
-      // This field is actually sitting on FragmentManagerImpl, which derives from FragmentManager.
-      if (mFieldMAdded == null) {
-        Field fieldMAdded =
-            ReflectionUtil.tryGetDeclaredField(fragmentManager.getClass(), "mAdded");
-
-        if (fieldMAdded != null) {
-          fieldMAdded.setAccessible(true);
-          mFieldMAdded = fieldMAdded;
+      if (fragmentManager instanceof android.app.FragmentManager) {
```

Comment:
The `instanceof` check here defeats the purpose of this abstraction a bit but this looks like a working fix otherwise.